File: solutions/006/006.php
<?php

$sumSquare = 0;
$squareSum = 0;
for ($i = 0; $i <= 100; $i++) {
    $squareSum += $i * $i;
    $sumSquare += $i;
}

echo $sumSquare * $sumSquare - $squareSum . "\n";

File: solutions/006/006.go
package main

import (
	"fmt"
)

func main() {
	var sumSquare, squareSum = 0, 0
	for i := 1; i <= 100; i++ {
		squareSum += i * i
		sumSquare += i
	}
	fmt.Println(sumSquare*sumSquare - squareSum)
}
